src/action.js

```diff
@@ -54,9 +57,13 @@ async function Main() {
     }
 }
 
-//API: ttps://octokit.github.io/rest.js/v18#repos-get-release-by-tag
+// Gets the Release for tag. The action will fail if the tag is not
+// found or if there is some other error getting the Release.
+//
+// GitHub API: https://octokit.github.io/rest.js/v18#repos-get-release-by-tag
 async function GetReleaseForTag() {
     core.info('GetReleaseForTag Start')
+
```

Comment:
if await octokit.repos.getReleaseByTag throws, then you won't get the corresponding core.info('GetLatestRelease Done') call. optionally consider try/finally